src/components/ScreenWrapper/index.native.js

```diff
@@ -0,0 +1,30 @@
+import {React, useEffect} from 'react';
+import DevMenu from 'react-native-dev-menu';
+import compose from '../../libs/compose';
+import toggleTestToolsModal from '../../libs/actions/TestTool';
+import {propTypes, defaultProps} from './propTypes';
+import BaseScreenWrapper from './BaseScreenWrapper';
+import withEnvironment from '../withEnvironment';
```

Comment:
I can't understand the usage of `withEnvironment` in both the components. If it is really required, then shouldn't it move to `BaseScreenWrapper` ?

Reply:
Yeah that looks better